packages/compass-signtool/src/sign.ts

```diff
@@ -0,0 +1,49 @@
+import path from 'path';
+import { exec } from 'child_process';
+import { promisify } from 'util';
+import { SSHClient } from './ssh-client';
+import { SigningClient } from './signing-client';
+import { assertRequiredVars } from './utils';
+
+const execAsync = promisify(exec);
+
+async function getSpawnnedHostName() {
+  // todo: this will fail
+  // return 'ec2-3-254-76-71.eu-west-1.compute.amazonaws.com';
+  const command = `tr -d '"[]{}' < ${path.resolve(
```

Comment:
i copied this from evergreen docs. but I do not intend to use this. 
following on your other comment on `privateKey`, in order to avoid this, i would pass these both options to the script. following options (initially):
1. host
2. username
3. port
4. privateKey (path)